inc/thorvg.h

```diff
@@ -319,6 +320,9 @@ class TVG_EXPORT Scene final : public Paint
 
     Result load(const std::string& path) noexcept;
     Result load(const char* data, uint32_t size) noexcept;
+    Result save(const std::string& path) noexcept;
+
+    void serialize();//MGS - temp?
```

Comment:
i'm not a big fun of adding this function, but I don't see how to code it in a different way...
I need this to start the serialization process in the run function (from task), because all serialization() functions are in Impls, so I need to have some api. 

I considered to use save - I could check in its body whether saver is null - then  open it (as it is happening right now), but if saver != null (now I simply close it and reopen) I could call the serialize() from Impl. But this solution doesn't seem to be elegant either....